AutoCollection/ServerRequests.ts

```diff
@@ -151,7 +151,7 @@ class AutoCollectServerRequests {
         var requestParser = _requestParser || new ServerRequestParser(request, correlationContext && correlationContext.operation.parentId || Util.newGuid());
 
         if (Util.canIncludeCorrelationHeader(client, requestParser.getUrl())) {
```

Comment:
Can you please ping @KamilSzostak  then and agree to what's the right thing to do. I think it's accidentally left out and should be fixed.